neurobooth_os/tasks/task_passage_reading.py

```diff
@@ -20,16 +18,16 @@ class Passage_Reading(Eyelink_HostPC):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+    def render_image(self):
+        self._render_image(op.join(neurobooth_os.__path__[0], "tasks/assets/bamboo_screenshot.jpg"),
+                          0, 0, 1920, 1080, 0, 0)
```

Comment:
Minor update to add kwarg names for clarity when reading? E.g., `crop_x=0, crop_y=0`

Reply:
Thanks Brandon. I merged this branch a minute before you posted your review. I'll make the change you suggested in a later PR. The crop_x, crop_y behavior is also weird, which I need to test and clarify in the documentation, and I have to add additional documentation regarding the bamboo image and how it is redered by psychppy on Anoopum's request. Also need to pass the image file paths as a yaml param instead of being hardcoded, so I'll bundle all those changes in the next PR.